lms/djangoapps/course_api/tests/test_serializers.py

```diff
@@ -0,0 +1,67 @@
+"""
+Test data created by CourseSerializer
+"""
+
+from datetime import datetime
+
+from rest_framework.test import APIRequestFactory
+from rest_framework.request import Request
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.course_module import DEFAULT_START_DATE
+
+from ..serializers import CourseSerializer
+from .mixins import CourseApiFactoryMixin
+
+
+class TestCourseSerializerFields(CourseApiFactoryMixin, ModuleStoreTestCase):
```

Comment:
My understanding was that SharedModuleStoreTestCase would create a course at the class level which would then be available for all the tests.  These tests are ones that need different configurations of courses for each test, so I want them destroyed after each individual test case.